lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -118,15 +180,14 @@ class window.Turbolinks
     triggerEvent 'page:receive'
     options.updatePushState ?= true
 
-    if doc = processResponse(xhr, options.partialReplace)
-      reflectNewUrl url if options.updatePushState
-      nodes = changePage(extractTitleAndBody(doc)..., options)
-      reflectRedirectedUrl(xhr) if options.updatePushState
+    doc = processResponse(xhr, options.partialReplace)
```

Comment:
Need to account for asset fingerprinting somewhere in here.  If:
* `admin/index-aaaaaaaaaaaaaaaaaaa.js` is in the page
* ... and the following page returns a head with `admin/index-bbbbbbbbbbbbbbbb.js`
* ... then a full refresh is necessary

 Otherwise we're breaking `data-turbolinks-track`.